source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -17,12 +17,14 @@ namespace JwtAuthn {
 namespace {
 
 Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>
-    access_control_request_method(Http::Headers::get().AccessControlRequestMethod);
+    access_control_request_method_handle(Http::CustomHeaders::get().AccessControlRequestMethod);
+Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>
+    origin_handle(Http::CustomHeaders::get().AccessControlRequestMethod);
```

Comment:
@mattklein123 shouldn't this be `Http::CustomHeaders::get().Origin`?

Reply:
I will send out a pr to fix this typo

Reply:
https://github.com/envoyproxy/envoy/pull/11911